caiman/base/movies.py

```diff
@@ -570,6 +571,57 @@ def crop(self, crop_top=0, crop_bottom=0, crop_left=0, crop_right=0, crop_begin=
         """
         t, h, w = self.shape
         self[:, :, :] = self[crop_begin:t - crop_end, crop_top:h - crop_bottom, crop_left:w - crop_right]
+    
+
+           
+    
+    def removeBL(self, windowSize:int=100, quantilMin:int=8, in_place:bool=False, returnBL:bool=False):                   
```

Comment:
There is a notation inconsistency here and in other places. Quantiles are between 0 and 1, but you use them to name percentile variables.